backend/services/admin.go

```diff
@@ -0,0 +1,52 @@
+package services
+
+import (
+	"backend/config"
+	"backend/models"
+	"backend/stores"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+type (
+	AdminService interface {
+		GetAdminById(id string) (*models.Admin, error)
+		GetAdminByUsername(username string) (*models.Admin, error)
+		CreateAdmin(creds *config.BasicAuth) (string, error)
+		// UpdateAdminById(user *models.User) (string, error)
+		// DeleteAdmin(id string) error
+	}
+
+	adminService struct {
+		stores *stores.Stores
+	}
+)
+
+func (s *adminService) CreateAdmin(creds *config.BasicAuth) (string, error) {
+	encryptedPassword, err := bcrypt.GenerateFromPassword(
+		[]byte(creds.Password),
+		bcrypt.DefaultCost,
+	)
+	if err != nil {
+		return "", err
+	}
+
+	admin := models.Admin{}
+	admin.PasswordHash = string(encryptedPassword)
+	admin.Username = creds.Username
```

Comment:
[want]
初期化はできるだけ同時にする方が簡潔で好みです。
```suggestion
	admin := models.Admin{
		admin.PasswordHash = string(encryptedPassword)
		admin.Username = creds.Username
	}
```

Reply:
ありがとうございます！修正完了です！